content/pages/changelog/index.md

```diff
@@ -13,6 +13,267 @@ For historical changelog entries, please see our
 [2019](page://changelog/2019) and
 [2018 release notes](page://changelog/2018).
 
+## 2020-06-11 / New enterprise events for tasks
+
+Starting today, the [enterprise event stream](g://events/for-enterprise/) will
+begin producing new task and task assignment events, and some existing task
+events will return additional fields.
+
+### New task and task assignment events
+
+#### Task update
+
+```js
```

Comment:
Might it be worth, for the sake of style, to add a little explanation before the code sample to say "This event happens when...."? Otherwise it feels like we just drop into the code samples.

Reply:
Can you use the suggest feature for these smaller change requests?

Reply:
I think this is no longer relevant. See other comments.